src/llmq/quorums_dkgsession.cpp

```diff
@@ -119,12 +115,17 @@ bool CDKGSession::Init(const CBlockIndex* _pindexQuorum, const std::vector<CDete
         }
     }
 
+    CDKGLogger logger(*this, __func__);
+
+    if (mns.size() < params.minSize) {
```

Comment:
Why this was moved?

Reply:
Because I wanted to use `logger` to print the new error message below. Compared to a simple `LogPrint`, this has the advantage that some additional LLMQ info is logged as well.

Reply:
So, this will partly initialize the session and then abort, but IMHO that's fine.